src/ImGui.NET/ImGuiNative.cs

```diff
@@ -195,23 +191,13 @@ public static unsafe class ImGuiNative
         [DllImport(cimguiLib)]
         public static extern void igDummy(Vector2* size);
         [DllImport(cimguiLib)]
-        public static extern void igIndent();
+        public static extern void igIndent(float indent_w);
```

Comment:
It should have a zero default (I sometimes miss these, because cimgui drops defaults). It was parameterless before 1.49.